static/js/common.js

```diff
@@ -0,0 +1,57 @@
+function getFacetsCheckboxes(collisionType, severity, noOfLanes) {
+        collisionType.length = 0 ;
```

Comment:
How about just use facetObj as the input parameter? Similar with the getFacetsRadiobuttons